Assets/MRTK/SDK/Profiles/DefaultMixedRealityControllerMappingProfile.asset

```diff
@@ -15,79 +15,53 @@ MonoBehaviour:
   isCustomProfile: 0
   mixedRealityControllerMappings:
   - controllerType:
-      reference: Microsoft.MixedReality.Toolkit.WindowsMixedReality.Input.WindowsMixedRealityArticulatedHand,
-        Microsoft.MixedReality.Toolkit.Providers.WindowsMixedReality
-    handedness: 1
+      reference: Microsoft.MixedReality.Toolkit.XRSDK.Input.OculusXRSDKRemoteController,
+        Assembly-CSharp
```

Comment:
You may need to reserialize this one as well, to update it with the new assembly